MSDOS/readme_mods.txt

```diff
@@ -0,0 +1,104 @@
+From:  Paul Baxter
+
+email: paje@globalnet.co.uk
+
+
+PURPOSE
+
+Put together a few batch files for the various versions I have.
+
+dealer.exe    - cygwin release version  Use cygwin.bat
+mdealer.exe   - MSDOS release version	  Use dostest.bat
+mddealer.exe  - MSDOS debug version     Use ddostest.bat
+
+Copy any of the Descr.* example files into this (MSDOS) directory.
+The batch files will run through each Descr.* file in turn.
+
+I thought this would be a crude initial step towards ensuring
+new versions produce same results as old, or that different computer
+systems produce similar results.
+
+Maybe use this test dir with examples and appropriate results to 'validate'
+new versions.
+
+I would prefer that the tests were in a separate dir to the MSVC project files
+but that's up to Henk. When I refer to the 'test' dir it is currently
+dealer/MSDOS dir
+
+
+
+USAGE
+   
+For the dos programs I have batch files dostest and ddostest (extra d for debug)

```

Comment:
I know what this means and it's benign, but naming something a "DDoS Test" feels scary :wink: